opentreemap/treemap/templates/instance_base.html

```diff
@@ -158,9 +161,14 @@
            disabled="disabled"><i class="icon-plus"></i> {% trans "Add a" %} {{ term.Resource.singular }}</a>
         {% endif %}
       <a class="btn btn-primary addBtn"
-         data-action='addtree'
+         {% if embed %}
+         target="_blank"
+         {% else %}
          data-feature="add_plot"
```

Comment:
`data-action` is still present, below, line 171.

`data-feature` is there for controlling add tree mode, which embed disables, and hiding the add tree control if save returns a result with `enabled` set to `False`. It seems unnecessary at best, and removing it helps assure that none of these actions will be triggered.

`data-event-category` and `data-event-action` trigger google analytics, which will be triggered when the new tab or window opens in `add` mode. We probably do not want to trigger them twice.

For reasons unknown to me, "Add a Resource" doesn't have these attributes to begin with.

Exactly what change are you proposing?